test/evergreen.yml

```diff
@@ -978,6 +978,25 @@ functions:
           ./t $format_args ${extra_args}
           ./t -R $format_args ${extra_args}
         done
+  "format test disagg":
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/cmake_build/test/format"
+      shell: bash
+      script: |
+        # To make sure we have plenty of flush_tier calls, we set the flush frequency high
```

Comment:
Comment is redundant -- looks like it came from the tiered storage version?

Reply:
Fixed